library/ssl_cache.c

```diff
@@ -311,8 +311,12 @@ int mbedtls_ssl_cache_set( void *data,
         ret = 1;
 #endif
 
-    if( session_serialized != NULL )
+    if( session_serialized != NULL ) 
```

Comment:
check_files is compaining about trailing whitespace here.

Reply:
Fixed.